### PR TITLE
Extension of the mouseClick function (add proportion and offset)

### PR DESCRIPTION
**Pull request by 53845714nF**
Add new parameters (proportion and offset) to the mouseClick function. For the end User this will end in two new Functions `mouseClickWithProportion` and `mouseClickWithOffset`.  It will be possible to click e.g. sliderarea at certain positions.

Can be used in Python, for example, as follows:
```python
from xmlrpc.client import ServerProxy

session = ServerProxy('http://localhost:9000')
session.mouseClickWithProportion('mainWindow/sliderarea', 0.4, 0.5)
```

### CODE REVIEW

**Comment by faaxm** [comment]
Thank you for your PRs @53845714nF ! I just want you to know that I am aware of them, but that I am super busy at the moment and it might take a while until I can have a proper look at them.

**Comment by 53845714nF** [comment]
No problem, I already suspected that. 
I'm happy that it was noticed. I will take a look at the clang error.

Happy Easter holidays.

**Comment by 53845714nF** [comment]
I hope I solve the Clang Error.

File: examples/GTestMouseClickPosition/main.cpp [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+    auto expected_result = R"RSLT(Button 1 clicked
+Button 2 clicked
+Button 3 clicked
+Button 4 clicked)RSLT";
+
+    EXPECT_EQ(result, expected_result);
+}
+
+TEST(GTestExample, ToMuchProportion)
```
Comment (faaxm) [inline examples/GTestMouseClickPosition/main.cpp:77]: it's supposed to be "too much", right?
```suggestion
TEST(GTestExample, TooMuchProportion)
```

**Review by faaxm — CHANGES_REQUESTED** [review]
Looks good 👍.
Could you merge in your pipeline fixes to stop the ci pipeline from failing? Once that runs through it can be merged :)

**Comment by faaxm** [comment]
just closing and reopening to have github rebase on latest master

**Comment by 53845714nF** [comment]
I renamed it.

**Review by faaxm — APPROVED** [review]
Nice and thanks for putting in the effort 👍 Hope I soon find time for the remaining reviews... Doing this purely in my free time, so unfortunately spix doesn't always make it to the top of my priority list.

**Pull request merged**